### ISSUE TITLE
Ability to customize snippets

### ISSUE THREAD

**Issue by emue-01** [sylvainjule/kirby-footnotes#10]
Hi @sylvainjule , I have tried to customize the snippets by placing them in my site/snippets folder as it works with most plugins. It doesn't seem to be working there, am I missing something? (I wanted to improve accessibility of the output).

**Comment by sylvainjule** [comment]
> The plugin's snippet declaration overrides any snippet placed in the site/snippets folder, I will add a way to register custom snippet names.

I just tried it and the plugin's declaration doesn't take precedence of the `site/snippets` ones, at least in Kirby 5 it works by placing snippets in this folder with the same names than the ones used by the plugin. Snippets have to be at the root of the folder though, that might have been the issue on your installation? I have added a way to register custom names / paths for snippets anyway.

**Comment by emue-01** [comment]
I've tried adding a custom `entry.php` file in `site/snippets` but it does not override the plugin's snippet. I will try it on a clean plainkit install when I find the time.
Accessibility improvements by @ffoodd meet my requirements, except for the hard coded value of the "back to content" link. I'm not sure what would be the right approach for this: maybe a config option allowing choosing between custom translation and custom text?

**Comment by sylvainjule** [comment]
I have added the translations + [an option](https://github.com/sylvainjule/kirby-footnotes?tab=readme-ov-file#53-back-title) to override it.
